src/Compilers/Core/Portable/Compilation/TypeInfo.cs

```diff
@@ -8,7 +8,7 @@ namespace Microsoft.CodeAnalysis
 {
     public struct TypeInfo : IEquatable<TypeInfo>
```

Comment:
readonly struct?

Reply:
I certainly could change this, but it feels outside the scope of this change.

I'm also not familiar with the API ramifications, are is that a breaking change?

---
In reply to: [259052143](https://github.com/dotnet/roslyn/pull/33531#discussion_r259052143) [](ancestors = 259052143)

Reply:
This is not a breaking change

---
In reply to: [259056221](https://github.com/dotnet/roslyn/pull/33531#discussion_r259056221) [](ancestors = 259056221,259052143)